### PR TITLE
Frontier Guns From The Artisans

### PR DESCRIPTION
**Pull request by bombasticSlacks**
#### Summary
Content "Adds a few hybrid melee/ranged weapons that the Artisans can craft"

#### Purpose of change
The reward from completing both artisan quests I've been putting off adding.  Combines firearm, and machining expertise with a lot of engineering to make some interesting new "frontier" style weapons.

#### Describe the solution
Some infrastructure:

- Sweet spot now makes noise based on the ammo type (.22 blanks)
- Spawns for .22 blanks in hardware stores
- Spell actions can now be "mundane" meaning they won't refer to casting spells but activating the item

Cody will request specific items, materials and cash to create the following projects over the course of 1 - 2 weeks.

Adds to the game:
**the Obrez**
image
Historically a weapon used as a makeshift pistol in places where it's easier to get a surplus rifle than a handgun. This version inspired by a popular cowboy themed shooter is reinforce with steel/iron, with an added finger catch at the end of the handguard, making it heavy but also a functioning mace.
image

TLDR: Mace + Pistol Mosin
Takes 1 week, 50 merch, 1 mosin

**the Pressin**
image
Historically a weapon used for self defense for VIP's and hidden in a glasses case. This .32 firing odd gun will also have a mounted switch blade and a slightly bolstered safety.

image

TLDR: Holdout 32 Pistol + Knife
Takes switch blade, .32 pistol, 20 merch 

**the Pulverizer**
image
+
image
Inspired visually by the Pulverize from Killing Floor but powered by powder charge blanks (used in industrial tools). This is a sledgehammer with a central piston that can be fired at high velocity, so if loaded it can deal huge crits.  IRL concept from here https://youtu.be/B3CsOx5U9Gs and https://youtu.be/Puo6Vgcbxps and finally here https://youtu.be/u7JnoRy2vng

image

TLDR: Sledgehammer + Really huge crits (I'm adding the code for this one in a separate PR)
Takes 2 weeks, sledghammer, 20 .22 blanks, 80 merch

**the Louverture**
This is the least scientifically grounded of these. A vented mace with an ignition that can be loaded to create a single, or perhaps based on balance/logistics, a few large puffs of fire a single tile in front of you for conflagration. It will be more awkward than a regular mace with the fuel reserve and otherwise be good for starting fires on just about anyone or anything quickly. I don't have a pic for this one...
image

TLDR: A mace + faux magic wand (if it only casts start fire)
Takes mace, flamethrower, 2 weeks

#### Describe alternatives you've considered
I may add more of these if the current ones go through.

#### Testing
Still WIP, basically just PRing so I can link this in my Pulverizer code.

### CODE REVIEW

**Comment by TheShadowFerret** [comment]
> Historically a weapon used for self defense for VIP's and hidden in a glasses case. This .32 firing odd gun will also have a mounted switch blade and a slightly bolstered safety.

I presume you know the caliber for that was a shortened .32 ACP, and the historical Pressin can't use actual .32 ACP.

**Comment by bombasticSlacks** [comment]
> I presume you know the caliber for that was a shortened .32 ACP, and the historical Pressin can't use actual .32 ACP.

It seemed overly convoluted to require custom modified ammo.

**Comment by AmarinReyny** [comment]
Is there an inspiration for The Louverture? I've had a similar sort of idea for a TTRPG I'm on-and-off "working on," and I'd like to know how others have imagined it, if so.

**Comment by bombasticSlacks** [comment]
This one was engineered a bit backwords but to explain in full: In the case of the Louverture it started with the prompt "bring the sun" which is a reference to the expiremental rock band Swans. It's not referenced much yet in game but Cody grew up/went to school in NYC and would be into a lot of post punk music.

So her weapons that aren't historical are going to all be references to new wave, no wave etc. And historical are going to have been used by revolutionaries. The Louverture ends up being a reference to both. 

Visually I drew inspiration from Sienna from Vermentides flamethrower staves https://vermintide2.fandom.com/wiki/Sienna%27s_Flamestorm_Staff/Illusions

**Comment by AmarinReyny** [comment]
Ah, I see... The idea I had was more like a combination of a mace and a fire lance, basically where you could load the mace head with a powder charge (rather than gasoline) and have it cause a small explosion/burst of fire from vents in the mace head (pointed outward, of course, so the wielder doesn't get blasted). I'm not sure how the powder would be ignited, though; I want it to have some kind of "trigger" near where it's gripped that could somehow create a spark where the powder is, but I have no idea how to incorporate that without rendering the weapon unusable as a mace.

File: data/json/items/gun/artisan_guns.json [hunk 1]
```diff
@@ -1,4 +1,30 @@
 [
+  {
+    "id": "obrez",
+    "copy-from": "mosin91_30",
+    "type": "GUN",
+    "name": { "str_sp": "the Obrez" },
+    "description": "A cut down modified Mozin-Nagant.  It has a short barrel and the stock has been cut down and fitted with a mace head.  The sides of the firearm as well have been reinforced with metal to withstand a beating.",
```
Comment (Faalagorn) [inline data/json/items/gun/artisan_guns.json:7]: Mosin
